docs/source/user_guide/data_actions.ipynb

```diff
@@ -242,27 +243,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Data check output returns a list of the dictionary version of each action that should be taken to clean up the data\n",
     "results[1]['actions']"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "We note that there are four action tasks that we can take to clean the data. Three of the tasks ask us to drop a row or column in the features, while one task asks us to impute the target value. "
-   ]
-  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# The first action states to drop the row given by the action code\n",
-    "X_train.drop(1477, axis=0, inplace=True)\n",
-    "# we must also drop this for y since we are removing its associated feature input\n",
-    "y_train.drop(index=1477, inplace=True)\n",
-    "\n",
+    "from evalml.pipelines.utils import make_pipeline_from_actions\n",
```

Comment:
I think we can now get rid of the `In the future, we aim to provide a helper function to allow users to quickly clean the data by taking in the list of actions and creating an appropriate pipeline of transformers to alter the data` line below?